src/main/flight/pid.c

```diff
@@ -1015,9 +1014,10 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
                 pidRuntime.itermAccelerator = 0.0f; // no antigravity on yaw iTerm
             }
         }
-
-        pidData[axis].I = constrainf(previousIterm + (Ki * dynCi + pidRuntime.itermAccelerator) * itermErrorRate, -pidRuntime.itermLimit, pidRuntime.itermLimit);
-
+        {
+        float iTermChange = (Ki + pidRuntime.itermAccelerator) * dynCi * pidRuntime.dT * itermErrorRate;
+        pidData[axis].I = constrainf(previousIterm + iTermChange, -pidRuntime.itermLimit, pidRuntime.itermLimit);
+        }
```

Comment:
We don't need brackets here?

```suggestion
        float iTermChange = (Ki + pidRuntime.itermAccelerator) * dynCi * pidRuntime.dT * itermErrorRate;
        pidData[axis].I = constrainf(previousIterm + iTermChange, -pidRuntime.itermLimit, pidRuntime.itermLimit);
```

Reply:
It looks a bit weird, but the brackets create an unnamed scope, so that the `iTermChange` variable that I added doesn't clutter the rest of the name space in this monster function. That said, they aren't strictly necessary and I'm fine with removing them if that is preferred.

Reply:
Like to have more opinions here @ledvinap ?

Reply:
The bracket is a drop in the ocean here. To make it a bigger drop in the ocean, how about this compromise: moving down the initializaiton of dynCi right to where it's needed and make an unnamed scope for that?

```c
        {
            // iTerm windup (attenuation of iTerm if motorMix range is large)
            float dynCi = 1.0f;
            if (pidRuntime.itermWindupPointInv > 1.0f) {
                dynCi *= constrainf((1.0f - getMotorMixRange()) * pidRuntime.itermWindupPointInv, 0.0f, 1.0f);
            }

            float iTermChange = (Ki + pidRuntime.itermAccelerator) * dynCi * pidRuntime.dT * itermErrorRate;
            pidData[axis].I = constrainf(previousIterm + iTermChange, -pidRuntime.itermLimit, pidRuntime.itermLimit);
        }
```

Reply:
The bracket is a drop in the ocean here. To make it a bigger drop in the ocean, how about this compromise: moving down the initializaiton of dynCi right to where it's needed and make an unnamed scope for that?

```c
        {
            // iTerm windup (attenuation of iTerm if motorMix range is large)
            float dynCi = 1.0f;
            if (pidRuntime.itermWindupPointInv > 1.0f) {
                dynCi *= constrainf((1.0f - getMotorMixRange()) * pidRuntime.itermWindupPointInv, 0.0f, 1.0f);
            }

            float iTermChange = (Ki + pidRuntime.itermAccelerator) * dynCi * pidRuntime.dT * itermErrorRate;
            pidData[axis].I = constrainf(previousIterm + iTermChange, -pidRuntime.itermLimit, pidRuntime.itermLimit);
        }
```

Reply:
Moving `dynCi` down is great for code readability, but unless the compiler does something really clever 
```C
dynCi *= constrainf((1.0f - getMotorMixRange()) * pidRuntime.itermWindupPointInv, 0.0f, 1.0f);
```
will be calculated 3 times instead of 1. If you think the performance impact is worth it I would happily implement this change. 

I also just realized that either way the `*=` can be replaced with a `=` now.

Reply:
Good point... the function is already so huge, we should try to stay efficient and keep dynCi outside of the loop, just to be sure.

Reply:
New version without braces pushed